README.md

```diff
@@ -279,9 +279,140 @@ In this case a directory named `4-procs` containing the simulation results outpu
 <!---                             Docker container                                        -->
 <!--- =================================================================================== -->
 
-<h1 id="docker_container">  Docker Container </h1>
-
-
+<h1 id="docker_container">  Docker </h1>
+An alternative way with respect to building from source, is the Docker option. To use this option Docker must be installed first. Please refer to [Docker webpage](https://www.docker.com/products/docker-desktop/) to know more about Docker and how to install it on your machine. The following steps describe how to build a Docker image or pull an existent one from DockerHub, and how to run a Docker container. The last section is a brief guide to perform the same steps but in Singularity, since HPC systems usually use Singularity to handle containers.
+
+## Docker image
+A Docker image is a read-only template that may contain dependencies, libraries, and everything needed to run a program. It is like a snapshot of a particular environment. 
+A Docker image can be created directly from a [dockerfile](https://docs.docker.com/reference/dockerfile/#:~:text=A%20Dockerfile%20is%20a%20text,can%20use%20in%20a%20Dockerfile.) or an existent image can be pulled from [DockerHub](https://hub.docker.com), if available. For this repository, both options are available.
```

Comment:
I think users could get confused that they might have to create their own docker file. I would move anything related to that to the `Docker` folder. I would keep it to the essentials in the main readme on how to use the provided docker files. You could distinguish between pulling the readily up-to-date svFSIplus container that you recently introduced (preferred for anyone just wanting to USE the code) and pulling only the externals and then compiling yourself (for anyone developing the code themselves).